iaso/api/groups/views.py

```diff
@@ -221,3 +148,74 @@ def dropdown(self, request, *args):
 
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data)
+
+    @action(detail=False, methods=["GET"], serializer_class=GroupExportSerializer)
+    def export(self, request, *args, **kwargs):
+        queryset = self.get_queryset()
+        filtered_queryset = GroupListFilter(request.query_params, queryset=queryset).qs
+        ordered_queryset = filtered_queryset.distinct("id").order_by("id")
+
+        serializer = self.get_serializer(data=request.query_params)
+        serializer.is_valid(raise_exception=True)
+
+        file_format = serializer.validated_data.get("file_format", None)
```

Comment:
`None` is not possible at this point.

So I'd be more strict here with e.g. `serializer.validated_data["file_format"]`.